bindata/network/multus-admission-controller/admission-controller.yaml

```diff
@@ -0,0 +1,63 @@
+---
+kind: DaemonSet
```

Comment:
Does this need to run on all nodes? I would rather either be a Deployment with a reasonable number of replicas (3?), or a DaemonSet only on the control plane.

Reply:
As the conclusion of OpenShift plumbing meeting, we decide to run admission controller for all master nodes (for HA). So DaemonSet + master toleration could be possible.

Could you please let me know how to run Pod for all master node (that will be configured so we cannot use '3' as fixed number) with ReplicaSet, if you know about it? I will follow you.